precompiles/assets/assets.go

```diff
@@ -89,12 +86,26 @@ func (p Precompile) Run(evm *vm.EVM, contract *vm.Contract, readOnly bool) (bz [
 	// It avoids panics and returns the out of gas error so the EVM can continue gracefully.
 	defer cmn.HandleGasError(ctx, contract, initialGas, &err)()
 
-	if method.Name == MethodDepositTo {
-		bz, err = p.DepositTo(ctx, evm.Origin, contract, stateDB, method, args)
+	switch method.Name {
+	// transactions
+	case MethodDepositTo, MethodWithdraw:
+		bz, err = p.DepositAndWithdraw(ctx, evm.Origin, contract, stateDB, method, args)
```

Comment:
Unify all "DepositAndWithdraw" instances into "DepositOrWithdraw."

Reply:
okay

Reply:
it will be resolved in the PR regarding registering tokens and client chains.